### ISSUE TITLE
Consider using "Manage" or "Configure" instead of "Administer"

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3845]
I have asked the following on Gitter as well:

> Is it only me that imagines doctors/nurses each time I see the verb "administer" being used in the admin UI?

![BECFE-administer](https://user-images.githubusercontent.com/2423362/58763761-91deba80-85a2-11e9-8d5d-01a17dbc9b79.png)

...I have always associated this word with medical professions:

![Screen Shot 2019-06-03 at 1 55 34 am](https://user-images.githubusercontent.com/2423362/58763788-dc603700-85a2-11e9-93f1-e838ead596e6.png)

**Comment by klonos** [comment]
...I am actually so annoyed by it, that I would like to see it being eradicated from the admin UI.

PS: for some weird reason, I am perfectly OK with the noun "administration" (although I'd prefer the short version for it in things like "admin UI" or "Admin Bar") ...it's just the verb that annoys me.

**Comment by klonos** [comment]
PPS: and don't anybody dare make any joke about the word "bar" in "Admin Bar" 😛

**Comment by findlabnet** [comment]
But as we can see from your search results:

> 1) manage and be responsible for...

seems to be a right answer.

**Comment by klonos** [comment]
Yep, "manage" also has a potentially ambiguous meaning of "survive" or "get by":

![Screen Shot 2019-06-03 at 3 48 25 am](https://user-images.githubusercontent.com/2423362/58765125-a165ff80-85b2-11e9-9fa4-fb9b764162a4.png)

"Configure" seems to be the least ambiguous of the 3:

![Screen Shot 2019-06-03 at 3 48 34 am](https://user-images.githubusercontent.com/2423362/58765129-b773c000-85b2-11e9-90f5-edb7ca3ae063.png)

**Comment by findlabnet** [comment]
Sorry for misunderstanding, I meant "administer" is historical right word, for example: whats wrong in "system administration"?

**Comment by klonos** [comment]
Yep, as I said, it's the verb "administer" I have issues with - not the noun:

>PS: for some weird reason, I am perfectly OK with the noun "administration" (although I'd prefer the short version of it in things like "admin UI" or "Admin Bar") ...it's just the verb that annoys me.
